components/DonatedItemsList.tsx

```diff
@@ -60,16 +75,33 @@ const DonatedItemsList: React.FC<DisplayDonatedItemsProps> = ({ userId }) => {
                   postable={item.postable}
                   itemId={item.id}
                 />
-                <div className='flex flex-row'>
+                <div className='flex flex-row gap-2'>
                   <Link href={`/edit-item/${item.id}`}>
-                    <p className='button button-rounded mx-2 my-2'>Edit item</p>
+                    <p className='button button-rounded my-2'>Edit item</p>
                   </Link>
                   <Modal
                     name='Delete Item'
                     itemId={item.id}
                     message='By pressing Confirm you will delete this item'
                     onDeleteSuccess={() => handleDeleteSuccess(item.id!)}
                   />
+                  {item.is_reserved ? (
+                    <UnreserveButton
+                      itemId={item.id}
+                      onReserveStatusChange={() =>
+                        onReserveStatusChange(item.id!)
+                      }
+                    />
```

Comment:
We maybe don't really need this component I think. This can be done with our `RoundedButton.tsx` component by just passing down the click handler 🤔 unless I'm overlooking something

Reply:
nice catch, I didn't even look at that component before. UI doesn't match but it works perfectly 🙏